src/rust/Dockerfile

```diff
@@ -110,14 +110,12 @@ RUN mkdir -p /grapl/zips; \
 
 # images for running services
 ################################################################################
-FROM debian:buster-slim AS rust-dist
+# More information about the base image used here can be found at: 
+# https://github.com/GoogleContainerTools/distroless/blob/main/cc/README.md.
+# For debugging see: https://github.com/GoogleContainerTools/distroless#debug-images
+FROM gcr.io/distroless/cc AS rust-dist
```

Comment:
I assume you tested, but this is indeed the minimal container for our current uses, correct?

Reply:
Yeah. We could probably get away with `static` if we statically linked to musl libc, and I'd tried with the `base` image, but we need libgcc, which is only found on the `cc` image. The description for this image even says it's suitable for Rust, so I'm confident this is the one. https://github.com/GoogleContainerTools/distroless/blob/main/cc/README.md